### PR TITLE
Add LICENSE file.

### PR DESCRIPTION
**Pull request by hatched**

### CODE REVIEW

**Comment by jujugui** [comment]
Refer to this link for build results (access rights to CI server needed): 
http://ci-cge.jujugui.org:8080//job/juju-gui/7219/

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,15 @@
+The Juju GUI is the graphical user interface used to interact with Juju.
+Copyright (C) 2017  Canonical Ltd.
```
Comment (jaycee) [inline LICENSE]: It it's the license file for the whole program, the (C) should probably be when it was started?

Dunno. Maybe the (C) is just for the file?

**Comment by hatched** [comment]
:shipit:

**Comment by jujugui** [comment]
Status: merge request accepted. Url: http://ci-gce.jujugui.org:8080/job/juju-gui-merge

**Pull request merged**